src/sentry/api/endpoints/project_details.py

```diff
@@ -251,12 +288,40 @@ def put(self, request, project):
         if result.get('digestsMaxDelay'):
             project.update_option(
                 'digests:mail:maximum_delay', result['digestsMaxDelay'])
-        if result.get('subjectPrefix'):
+        if result.get('subjectPrefix') is not None:
```

Comment:
Previously you wouldn't be able to remove a value, unsure if we want user to be able to delete some of these keys below.